src/mousetracker.js

```diff
@@ -1400,11 +1400,20 @@
          * @function
          */
         removeContact: function() {
-            --this.contacts;
-
-            if (this.contacts < 0) {
-                $.console.warn('GesturePointList.removeContact() Implausible contacts value');
-                this.contacts = 0;
+            if (
+                (this.type === "mouse" ||
```

Comment:
What other types are there besides mouse, pen, and touch? Do we need this part of the if statement?